pennylane_lightning/core/src/algorithms/tests/Test_AdjointJacobian.cpp

```diff
@@ -25,6 +25,7 @@ using namespace Pennylane::Util;
 
 #ifdef _ENABLE_PLQUBIT
 constexpr bool BACKEND_FOUND = true;
+constexpr bool WITH_CONTROLS = true;
```

Comment:
Nit-pick: not sure about the naming of this. Maybe we can think of something else along the way

Reply:
Maybe `SUPPORTS_CTRLS`?

Reply:
Sounds good to me.